Libraries/LibJS/Runtime/StringPrototype.cpp

```diff
@@ -368,4 +369,35 @@ Value StringPrototype::substring(Interpreter& interpreter)
     return js_string(interpreter, string_part);
 }
 
+Value StringPrototype::includes(Interpreter& interpreter)
+{
+    auto* this_object = interpreter.this_value().to_object(interpreter.heap());
+    if (!this_object)
+        return {};
+
+    if (interpreter.argument_count() == 0)
+        return Value(false);
```

Comment:
This is not correct.

`"string contains undefined".includes()` is supposed to return true.

> https://tc39.es/ecma262/#sec-string.prototype.includes
>
> 21.1.3.7 String.prototype.includes ( searchString [ , position ] )
> ...
> Let searchStr be ? ToString(searchString).